src/main/java/org/embulk/filter/calcite/PageConverter.java

```diff
@@ -81,10 +82,10 @@ public void timestampColumn(Column column) {
             row[i] = null;
         } else {
             // Embulk's timestamp is converted into java.sql.Timestmap
-            org.embulk.spi.time.Timestamp timestamp = pageReader.getTimestamp(i);
-            long milliseconds = timestamp.getEpochSecond() * 1000 + timestamp.getNano() / 1000000;
+            final Instant instant = getInstant(pageReader, i);
+            long milliseconds = instant.getEpochSecond() * 1000 + instant.getNano() / 1000000;
```

Comment:
i think this equal to `instant.toEpochMilli()`?

Reply:
I guess so, but please let me keep this as of now so that we can easily confirm that this is just a catch-up. Additional refactoring and cleanups would come after confirming the catch-up works.